contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -179,6 +178,11 @@ void SetResetTDSConnectionFlag()
 	resetTdsConnectionFlag = true;	
 }
 
+bool GetResetTDSConnectionFlag()
```

Comment:
Nit(non-blocking): Return type and function name in different lines.

Reply:
I dont see a PG guideline for this, codebase is inundated with single line declarations for such util functions